packages/component-submission/server/aggregates/Submission.test.js

```diff
@@ -11,88 +13,92 @@ const createMockObject = (values = {}, mockSaveFn) => ({
   save: mockSaveFn || jest.fn(),
 })
 
+const createSubmission = (args = {}) => {
+  const {
+    models = { Manuscript, File, Team },
+    services = { Storage: { getDownloadLink: jest.fn() } },
+  } = args
+
+  return new Submission({ models, services })
+}
+
 describe('Submission', () => {
+  let mockManuscriptFind, mockFileFind, mockTeamFind
+
+  beforeAll(() => {
+    mockManuscriptFind = jest.spyOn(Manuscript, 'find')
+    mockFileFind = jest.spyOn(File, 'findByManuscriptId')
+    mockTeamFind = jest.spyOn(Team, 'findByManuscriptId')
```

Comment:
these three mocks can be instantiated outside the beforeAll() where they are declared.